### PR TITLE
Add info on 2FA requirement

### PR DESCRIPTION
**Pull request by laurelmay**
We now require 2FA in order to be a member of the EasyDynamics GitHub
organization. This adds a note about that so that users who do not have
that configured already can be prepared to do so.

Various spelling issues and extraneous whitespace throughout the document
are also cleaned up.

### CODE REVIEW

File: index.md
Comment (rgauss) [inline index.md]: Should this 'S' be lower-case?

**Review by rgauss — APPROVED** [review]
Minor suggestion that doesn't need to hold up review.

File: index.md
Comment (laurelmay) [reply index.md]: Good catch!

**Comment by laurelmay** [comment]
Seeing as it's been ~1wk and this has gotten an approval and no objections, I am going to move forward with merging.

**Pull request merged**
